tiled/client/array.py

```diff
@@ -177,22 +177,24 @@ def read(self, slice=None):
             dask_array = dask_array[slice]
         return dask_array
 
-    def write(self, array):
+    def write(self, array, persist=True):
         for attempt in retry_context():
             with attempt:
                 handle_error(
                     self.context.http_client.put(
                         self.item["links"]["full"],
                         content=array.tobytes(),
                         headers={"Content-Type": "application/octet-stream"},
+                        params={"persist": bool(persist)},
```

Comment:
To keep the server logs as readable as possible, we have elsewhere adopted a pattern of only specifying optional params when they are not the default value, to cut down on clutter. I think it makes sense to apply that pattern here too (and in the other writing calls below) because the default value will very frequently be used.